### ISSUE TITLE
TOPICS: Users getting 'Join topic' or 'Join group' pop-up when no longer relevant

### ISSUE THREAD

**Issue by BeccaMelhuish** [citizenos/citizenos-fe#1284]
@ilmartyrk @kevincrepin Just a UX bug I noticed in the ideation user tests, probably doesn't need much discussion: 

Users are getting ‘Join topic’ or ‘Join group’ pop-up when following an invite/share link, even if they have already joined or even created it. 

Users often use the same link to get back to the topic, not just to join it. And admins will be posting the link on social media to get people to come to each next phase of the topic (when they have probably already participated in previous phases). So they should only get it the first time, when they are actually joining.

EDIT: Seems to only happen for topics not groups? Not sure though.

**Comment by BeccaMelhuish** [comment]
Will add here that a user also raised this as surprising - that every time she followed a link to join each phase of the process, she was asked to join, despite already having joined and contributed. 

She did acknowledge that it was likely bothering her more due to the fact we were 'speeding up time' and doing all the phases in quick succession.

**Comment by kevincrepin** [comment]
in these cases was the person also still logged in?

**Comment by BeccaMelhuish** [comment]
Yes still logged in, and a later participant tester also confirmed this was confusing and surprising, to be asked to join again every time. I think how it is set up currently is that if you follow a join link to a topic, then you get the 'join' pop-up, regardless of whether you've already joined.

**Comment by kevincrepin** [comment]
I think then it's a technical thing where the cookie or session or whatever it is should log that this person is logged in and has already joined the topic, so the dialog window should not be shown. Easier for me to say than to make it so of course! :)

**Comment by BeccaMelhuish** [comment]
I just tried this myself in live and it seems to be fixed, but will send to @ssin1901 for testing to be sure :)

**Comment by BeccaMelhuish** [comment]
Will close it for now though :)

**Comment by BeccaMelhuish** [comment]
Re-opening as it just happened again (following the link to our fortnightly meetings topic from our calendar invite)
